posthog/async_migrations/migrations/0004_replicated_schema.py

```diff
@@ -100,6 +100,8 @@ class Migration(AsyncMigrationDefinition):
 
     depends_on = "0003_fill_person_distinct_id2"
 
+    posthog_max_version = "1.36.1"
```

Comment:
let's maintain convention of having this follow minor releases

```suggestion
    posthog_max_version = "1.36.99"
```